ic-utils/src/interfaces/management_canister.rs

```diff
@@ -307,6 +337,65 @@ impl<'agent> Canister<'agent, ManagementCanister> {
             .build()
     }
 
+    /// Until developers can convert real ICP tokens to provision a new canister with cycles,
+    /// the system provides the provisional_create_canister_with_cycles method.
+    /// It behaves as create_canister, but initializes the canister’s balance with amount fresh cycles
+    /// (using MAX_CANISTER_BALANCE if amount = null, else capping the balance at MAX_CANISTER_BALANCE).
+    /// Cycles added to this call via ic0.call_cycles_add are returned to the caller.
+    /// This method is only available in local development instances, and will be removed in the future.
+    pub fn provisional_create_canister_with_cycles<'canister: 'agent>(
+        &'canister self,
+        amount: Option<u64>,
+    ) -> impl 'agent + AsyncCall<(Principal,)> {
+        #[derive(CandidType)]
+        struct Argument {
+            amount: Option<u64>,
```

Comment:
This should fix the issue:
```suggestion
            amount: Option<candid:Nat>,
```